custom/icds_reports/tests/test_fact_sheet_report.py

```diff
@@ -979,13 +979,13 @@ def test_cases_household(self):
             self.get_data()['config']['sections'][0]['rows_config'][0],
             {
                 'average': {
-                    'html': 6964,
-                    'sort_key': 6964
+                    'html': 2799,
+                    'sort_key': 2799
                 },
                 'data': [
                     {'html': 'Number of Households'},
-                    {'html': 6951, 'sort_key': 6951},
-                    {'html': 6964, 'sort_key': 6964},
+                    {'html': 2792, 'sort_key': 2792},
+                    {'html': 2799, 'sort_key': 2799},
```

Comment:
@Rohit25negi this is the thing I was surprised by related to [this comment](https://github.com/dimagi/commcare-hq/pull/25354#issuecomment-531198142)

Reply:
Yes @czue , These numbers dropped significantly because our test data condition is significantly different from what is in prod. For example, There is a case which is called "household case", if an awc has created that case ever that awc has to be launched and if the  case is never created awc can't be launched logically. So this drop of number is to make the test cases support that statement.
For this instance, Earlier in the test cases there were awcs which was marked not launched but they did have "household" case created. So, There were two ways I could Fix it. Either mark then all launched(which would have affected much more test cases changes than this) OR I deleted the "household" cases from those awcs(which is the what I did).

Does that clarifies the question? feel free to offline in case thats faster.

Reply:
:+1: thanks for clarifying